.github/workflows/tests.yaml

```diff
@@ -61,5 +62,11 @@ jobs:
                     dependency-versions: ${{ matrix.dependencies }}
                     composer-options: "--prefer-dist --prefer-stable"
 
+            -   name: Remove dependencies not compatible with PHP 8
```

Comment:
I think you should create a make test/php8 command instead in order to not run the mongo tests.
https://github.com/theofidry/AliceDataFixtures/blob/master/Makefile#L116

Reply:
Thanks for your help but I didn't want to duplicate all of these lines: https://github.com/theofidry/AliceDataFixtures/blob/22040ef893953a74f02f7c678cb2f14641cb7d60/Makefile#L80

And [`make test_symfony_doctrine_bridge_proxy_manager`](https://github.com/theofidry/AliceDataFixtures/blob/22040ef893953a74f02f7c678cb2f14641cb7d60/Makefile#L184) will call that [configuration](https://github.com/theofidry/AliceDataFixtures/blob/7165a10e4e794852ea50d9783873f7ee34f68516/phpunit_symfony_proxy_manager_with_doctrine.xml.dist#L28) and that will launch [several tests](https://github.com/theofidry/AliceDataFixtures/tree/f2c9e5432c9e0a61f2d4d6440c295a412951945e/tests/Bridge/Symfony/Doctrine), and only one require `doctrine/mongodb-odm-bundle`.

So I thought it was simpler to remove the dependency and skip the test that rewrite Makefile or PHPUnit configuration.

Reply:
It will be addressed in #168